src/main/java/net/schmizz/sshj/transport/KeyExchanger.java

```diff
@@ -40,7 +40,7 @@
 final class KeyExchanger
         implements SSHPacketHandler, ErrorNotifiable {
 
-    private static enum Expected {
+    private enum Expected {
```

Comment:
Not sure why this change is in there ;)

Reply:
Oups, I am sorry. I did it mechanically because IDEA highlighted it as redundant modifier (which is redundant indeed). Since it is certainly is unrelated changed, I have removed it :)